### DISCUSSION TITLE
Incomplete file

### DISCUSSION THREAD

**Discussion by alixiaodi** [minio/minio#19247] [Q&A]
1.How to view all incomplete file information
2. How to modify incomplete file deletion strategies

**Comment by klauspost** [comment]
1. https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListMultipartUploads.html

2. https://docs.aws.amazon.com/AmazonS3/latest/API/API_AbortMultipartUpload.html

https://min.io/docs/minio/linux/reference/minio-mc/mc-rm.html#remove-all-incomplete-upload-files-for-an-object

`mc admin config set ALIAS api stale_uploads_cleanup_interval=6h stale_uploads_expiry=24h`

**Reply by alixiaodi** [reply]
Can I make modifications through the console

**Reply by alixiaodi** [reply]
@klauspost

**Reply by klauspost** [reply]
@alixiaodi I do not know. I do know you can do it through mc.

**Reply by alixiaodi** [reply]
Ok, thank you.
